src/elements-experimental/timetable-row/timetable-row.ssr.spec.ts

```diff
@@ -13,7 +13,10 @@ describe(`sbb-timetable-row ssr`, () => {
     });
   });
 
-  it('renders', () => {
```

Comment:
I saw also another test with the same error: the journey summary ssr. do we need in general a slightly higher timeout for ssr tests?